tools/schemacode/bidsschematools/tests/test_make_testdata.py

```diff
@@ -0,0 +1,47 @@
+import os
+import shutil
+import tarfile
+import tempfile
+
+import pytest
+
+from .. import version_file
+
+
+def require_env(var):
+    env = os.environ.get(var)
+    return pytest.mark.skipif(
+        not env,
+        reason="Release mode is not activated. Not generating static testdata archive. "
+        "In order to force testdata archive generation run: "
+        "`export BIDSSCHEMATOOLS_RELEASE_MODE_ACTIVATED=1`.",
+    )
+
+
+@require_env("BIDSSCHEMATOOLS_RELEASE_MODE_ACTIVATED")
```

Comment:
I'd try to write this decorator a bit more generically, and then add a reason where called. And I'd also suggest a shorter environment variable:

```suggestion
def require_env(var, reason=None):
    instructions = f"To activate this test/feature `export {var}=1` and re-run."
    return pytest.mark.skipif(
        not os.getenv(var),
        reason=f"{reason} {instructions}" if reason else instructions
    )


@require_env("BST_RELEASE", "Release mode inactive; not generating test data archive.")
```